pycbc/workflow/core.py

```diff
@@ -1517,8 +1518,7 @@ def to_segment_xml(self, override_file_if_exists=False):
         # create XML doc and add process table
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
-        process = ligolw_utils.process.register_to_xmldoc(outdoc, sys.argv[0],
-                                                          {})
+        process = ligolw_process.register_to_xmldoc(outdoc, sys.argv[0], {})
```

Comment:
Hi Andrew,

I think I understand the problem now. Your example:

```
from glue.ligolw import utils as ligolw_utils
ligolw_utils.process
```

fails because process is not imported. *BUT* if some *other* file has already done:

```
import glue.ligolw.utils.process
```

then the above code works. So the original code required an import to have been made outside of the core.py module. Your original fix was the correct one, please revert the change e4f8e5c to bring this back to the version I looked at, and then this can be pushed.